.editorconfig

```diff
@@ -7,11 +7,13 @@ indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
+[*.{java,kt}]
 # Disable wildcard imports
-[*.{java, kt}]
```

Comment:
Needed to remove the whitespace between `java` and `kt`: https://pinterest.github.io/ktlint/0.50.0/rules/configuration-ktlint/

> Unfortunately [IntelliJ IDEA](https://www.jetbrains.com/idea/) has an [autoformat issue regarding .editorconfig](https://youtrack.jetbrains.com/issue/IDEA-242506). Due to this error an additional space is added between glob statements, resulting in [*{kt, kts}] instead of [*{kt,kts}]. The .editorconfig library used by ktlint [ignores sections after encountering a space in the list](https://github.com/editorconfig/editorconfig/issues/148). As a result, the rule is not applied on all files as documented in the [original ktlint issue](https://github.com/pinterest/ktlint/issues/762).

Reply:
Good catch, thanks.